GitVersionCore.Tests/IntegrationTests/GitFlow/GitFlowFeatureBranchTests.cs

```diff
@@ -17,7 +17,7 @@ public void ShouldNotUseNumberInFeatureBranchAsPreReleaseNumber()
             fixture.Repository.Checkout("feature/JIRA-123");
             fixture.Repository.MakeCommits(5);
 
-            fixture.AssertFullSemver("1.1.0-JIRA-123+5");
+            fixture.AssertFullSemver("1.1.0-JIRA-123.1+5");
```

Comment:
I think this new behaviour is correct. the .1 is the same as we put on -beta, -unstable etc.